indico/htdocs/js/custom/dropzone.js

```diff
@@ -94,6 +95,14 @@
                         if (options.submit_form) {
                             $form.submit();
                         }
+                        if (options.handleFlashes) {
+                            handleFlashes(response, true, $flashArea);
+                        }
+
+                        $dz.data('value', response.content);
+                        $form.find('.change-trigger').val($dz.data('value') ? 1 : 0);
```

Comment:
i'd quote the numbers since the value is stored as a string anyway. explicit > implicit ;)